src/modules/commander/state_machine_helper.cpp

```diff
@@ -56,7 +56,7 @@ static constexpr const char reason_no_offboard[] = "no offboard";
 static constexpr const char reason_no_rc_and_no_offboard[] = "no RC and no offboard";
 static constexpr const char reason_no_local_position[] = "no local position";
 static constexpr const char reason_no_global_position[] = "no global position";
-static constexpr const char reason_no_datalink[] = "no datalink";
+static constexpr const char reason_no_datalink[] = "no GCS datalink";
```

Comment:
How about ground control instead of GCS?

Reply:
I don't see a huge difference, but I can change it.